src/ext.rs

```diff
@@ -0,0 +1,9 @@
+pub trait OptionExt<T, E> {
```

Comment:
[nit] We should add the file header to new files.

Reply:
Addressed with 274aae29473ed2a40452621901ddb227d077fb0b.

We should have a CI check for this.